raft/raft.cpp

```diff
@@ -39,16 +40,18 @@ using namespace bzn;
 
 
 raft::raft(
-        std::shared_ptr<bzn::asio::io_context_base> io_context
-        , std::shared_ptr<bzn::node_base> node
-        , const bzn::peers_list_t& peers
-        , bzn::uuid_t uuid
-        , const std::string state_dir
-        , size_t maximum_raft_storage)
-    : timer(io_context->make_unique_steady_timer())
-    , uuid(std::move(uuid))
-    , node(std::move(node))
-    , state_dir(std::move(state_dir))
+        std::shared_ptr<bzn::asio::io_context_base> io_context,
+        std::shared_ptr<bzn::node_base> node,
+        const bzn::peers_list_t& peers,
+        bzn::uuid_t uuid,
+        const std::string state_dir, size_t maximum_raft_storage,
+        bool whitelist_enabled
+        ) :
```

Comment:
this should be : on a new line and comma's underneath it 

like:

)
: timer()
, uuid()
, node()

etc.

Reply:
Cool, I'll make the change.